org_fedora_oscap/rule_handling.py

```diff
@@ -668,3 +683,72 @@ def eval_rules(self, ksdata, storage, report_only=False):
             return []
 
     # nothing to be reverted for now
+
+
+class KdumpRules(RuleHandler):
+    """Simple class holding data from the rules affecting the kdump addon."""
+
+    def __init__(self):
+        """Constructor setting the initial value of attributes."""
+
+        self._kdump_enabled = None
+        self._kdump_default_enabled = None
+
+    def kdump_enabled(self, kdenabled):
+        """Enable or Disable Kdump"""
+
+        if kdenabled is not None:
+            kdenabled = self._kdump_enabled
+
+    def __str__(self):
+        """Standard method useful for debugging and testing."""
+
+        ret = "kdump"
+
+        if self._kdump_enabled == True:
+            ret += " --enable"
+
+        if self._kdump_enabled == False:
+            ret += " --disable"
+
+        return ret
+
+    def eval_rules(self, ksdata, storage, report_only=False):
+        """:see: RuleHandler.eval_rules"""
+
+        messages = []
+
+        if self._kdump_enabled == None:
```

Comment:
Please either change 
* all the `==` in this if..elif..else block to `is`, or
* just check for `None` using `is`, and shorten the subsequent checks - you can safely assume that `self._kdump_enabled is False` is the same as `not self._kdump_enabled` and ditto for the converse case.